pkg/pool-manager/virtualmachine_pool.go

```diff
@@ -433,6 +433,12 @@ func (p *PoolManager) MarkVMAsReady(vm *kubevirt.VirtualMachine, parentLogger lo
 			return nil
 		}
 
+		if len(vm.Spec.Template.Spec.Domain.Devices.Interfaces) == 0 {
```

Comment:
Why do we need this ?

Reply:
this intended to remove a configmap update if there is no real change made to it.

Reply:
We are already iterating we will not iterate if this is empty we don't need this pre-check